docs/spec_draft.md

```diff
@@ -825,6 +826,247 @@ Produces an `output` tensor from a constant `value`.
+  ##
+  ## Compute the components in lhs_full_index at spatial dimensions.
+  ##
+  i0' = result_index[output_spatial_dimension[0]] * window_strides[0] - padding[0][0] + k0 * rhs_dilation[0]
+  i0 = base_dilations[0] > 1 ? i0' / base_dilations[0] : i0'
+
+  i1' = result_index[output_spatial_dimension[1]] * window_strides[1] - padding[1][0] + k1 * rhs_dilation[1]
+  i1 = base_dilations[1] > 1 ? i1' / base_dilations[1] : i1'
+  ... omiting similar logic for other spatial dimensions ...
+
+  if i0' % lhs_dilation[0] != 0 or i0 not in bounds of dim(lhs, 0), then skip [k0, k1]
+  if i1' % lhs_dilation[1] != 0 or i1 not in bounds of dim(lhs, 1), then skip [k0, k1]
+  ... omiting similar logic for other spatial dimensions ...
+
+  lhs_full_index[input_spatial_dimension] = [i0, i1, ...]
+
+  ##
+  ## Compute the components in rhs_full_index at spatial dimensions.
+  ##
+  rhs_full_index[kernel_spatial_dimension] = [
+    window_reversal[0] ? dim(rhs, kernel_spatial_dimension[0]) - 1 - k0 : k0,
+    window_reversal[1] ? dim(rhs, kernel_spatial_dimension[1]) - 1 - k1 : k1
+    ... omiting similar logic for other spatial dimensions ...
+  ]
+
+  input_feature_group_size = dim(lhs, input_feature_dimension) / feature_group_count.
+  foreach rhs_iz in [0, input_feature_group_size)
+    ##
+    ## Compute the components in lhs_full_index at input_batch_dimension  and input_feature_dimension.
+    ##
+    lhs_full_index[input_batch_dimension] = result_index[output_batch_dimension] + ((batch_group_index * batch_group_size) % input_batch_size)
+      where
+        batch_group_size = input_batch_size / batch_group_count
+        input_batch_size = dim(lhs, input_batch_dimension)
+        batch_group_index = result_index[output_feature_dimension] / depthwise_multiplier
+        depthwise_multiplier = batch_group_count > 1 ? dim(rhs, kernel_output_feature_dimension) / batch_group_count : 1
+
+    lhs_full_index[input_feature_dimension] = rhs_iz + feature_group_index * input_feature_group_size
+      where
+        feature_group_index = result_index[output_feature_dimension] / output_feature_group_size
+        output_feature_group_size = dim(rhs, kernel_output_feature_dimension) / feature_group_count
+
+    ##
+    ## Compute the components in rhs_full_index at kernel_input_feature_dimension and kernel_output_feature_dimension.
+    ##
+    rhs_full_index[kernel_input_feature_dimension] = rhs_iz
+
+    rhs_full_index[kernel_output_feature_dimension] = result_index[output_feature_dimension]
+
+    ##
+    ## Accumulate value to be stored in result at 'result_index'.
+    ##
+    value += rhs[rhs_full_index] * lhs[lhs_full_index]
+
+  result[result_index] = value
+```
+
+`precision_config` controls the tradeoff between speed and accuracy for
+computations on accelerator backends. The values can be one of the followings:
+  * `DEFAULT`: Fastest mode, but least accurate (default value).
+  * `HIGH`: Slower, but more accurate.
+  * `HIGHEST`: Slowest, but most accurate.
```

Comment:
Actually, since we reformulated ConvolutionOp in terms of DotGeneralOp, let's just drop this paragraph and rely on the description of `precision_config` in DotGeneralOp.